db/migrate/20170319193517_add_badge_id_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddBadgeIdToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :badge_id, :integer
+    add_column :users, :badge_id, :integer, default: 0
+    User.where(donor: true).update_all(badge_id: 1)
```

Comment:
Could you add the Badge creations from the seeds file here as well? Seeds don't run on migrations, so this references a Badge that doesn't exist. You could then use `.update_all(badge: ref_to_badge)`